pipelines/nextflow/workflows/nextflow.config

```diff
@@ -114,4 +114,7 @@ process {
         storeDir = { "${params.cache_dir}/${task.process.tokenize(':')[-1].toLowerCase()}/${task.tag}" }
         afterScript = { "sleep ${params.storeDir_latency}" }
     }
+    withLabel: 'datasets_cli' {
+        container = 'library://lcampbell/ensembl-genomio/ncbi-datasets-v16.17.1:latest'
```

Comment:
Something we definitely need to replace soon with a proper URL for the sake of usability by other people.

Reply:
Sounds good, until then we stick with this hack I guess :)